ReleaseTooling/Sources/FirebaseManifest/FirebaseManifest.swift

```diff
@@ -44,10 +44,7 @@ public let shared = Manifest(
     Pod("FirebasePerformance", platforms: ["ios", "tvos"], zip: true),
     Pod("FirebaseStorage", zip: true),
     Pod("FirebaseStorageSwift", isBeta: true),
-    Pod("FirebaseMLCommon", isClosedSource: true, isBeta: true, platforms: ["ios"]),
-    Pod("FirebaseMLModelInterpreter", isClosedSource: true, isBeta: true, platforms: ["ios"],
-        zip: true),
-    Pod("FirebaseMLVision", isClosedSource: true, isBeta: true, platforms: ["ios"], zip: true),
```

Comment:
I think these should stay, although I could be convinced that we should stop publishing "new" versions